lib/webfuse/core/container_of.h

```diff
@@ -0,0 +1,13 @@
+#ifndef WF_CONTAINER_OF_H
+#define WF_CONTAINER_OF_H
+
+#ifndef __cplusplus
+#include <stddef.h>
+#else
+#include <cstddef>
+#endif
+
+#define WF_CONTAINER_OF(pointer, type, member) \
```

Comment:
It would be better to switch implementation depending on available compiler features